src/coord_seq.rs

```diff
@@ -96,7 +97,7 @@ impl<'a> CoordSeq<'a> {
             if !data.iter().skip(1).all(|x| x.len() == dims) {
                 return Err(Error::GenericError("All vec entries must have the same size!".to_owned()));
             }
-            match match GContextHandle::init() {
+            match GContextHandle::init() {
```

Comment:
can't you put and_then on the first one too ?

`GContextHandle::init().and_then( unsafe {...}).and_then(|coord|...`

Reply:
You're asking a bit too much of me! I love `match`es. :'(